src/vma/util/sys_vars.cpp

```diff
@@ -1053,8 +1066,14 @@ void mce_sys_var::get_env_params()
 		cq_poll_batch_max = MCE_DEFAULT_CQ_POLL_BATCH;
 	}
 
-	if ((env_ptr = getenv(SYS_VAR_PROGRESS_ENGINE_INTERVAL)) != NULL)
+	if ((env_ptr = getenv(SYS_VAR_PROGRESS_ENGINE_INTERVAL)) != NULL) {
 		progress_engine_interval_msec = (uint32_t)atoi(env_ptr);
+	}
+	if (enable_socketxtreme && (progress_engine_interval_msec != MCE_CQ_DRAIN_INTERVAL_DISABLED)) {
```

Comment:
We will have a warning whenever we will run spec latency with socketxtreme...
we cant do it

Reply:
There are a couple of environment variables that can conflict with some specs (keep in mind  that specs are flexible and can add/remove different variables) so it means that socketxtreme can not be used with these specs. Please advise an alternative.

Reply:
i understand. but latency spec is our recommended configuration for latency.
we should not print any warning. we should suppress it somehow.

Reply:
changed to LOG_DEBUG